src/amo/components/Addon/index.js

```diff
@@ -271,28 +270,23 @@ export class AddonBase extends React.Component {
           title = i18n.gettext('About this add-on');
       }
 
-      const description = addon.description ? addon.description : addon.summary;
-      // For any theme type, we want to hide the summary text here since that is
-      // already displayed in the header.
-      showAbout =
-        ((description === addon.summary &&
-          addon.type === ADDON_TYPE_STATIC_THEME) ||
-          addon.type === ADDON_TYPE_THEME) === false;
+      const { description } = addon;
 
       if (!description || !description.length) {
```

Comment:
Maybe we should also check that `description !== summary`, no? I am thinking about that because of the test case "does not display a lightweight theme's summary" you changed.

Reply:
Yeah I was thinking about why that test had changed.. I'll try it out later today when I get home from work!

Reply:
I tried adding `description !== summary` but 12 other tests broke and my gut instinct told me that wasn't probably the right route. Instead, I update that single test since `description` wasn't passed in at all - what are your thoughts?